java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/impl/SqlClusterExecutor.java

```diff
@@ -72,6 +83,22 @@ public SqlClusterExecutor(SdkOption option) throws SqlException {
         }
     }
 
+    public static String getSqlJsdkLibraryPath() {
```

Comment:
Should this function move to LibraryLoader class?

Reply:
Since we will alway find the path of "sql_jsdk", it is not suitable to add in `LibraryLoader`.